usr.sbin/syslogd/syslogd.8

```diff
@@ -249,6 +250,10 @@ the default is
 Specify an alternative file in which to store the process ID.
 The default is
 .Pa /var/run/syslog.pid .
+.It Fl R Ar rate_limit
+Specify a max bytes per minute to log.
+Useful to prevent /var/log from taking over your disk.
```

Comment:
This might be better:
<pre>
This helps mitigate filling up the partition where
.Pa "/var/log"
exists.
</pre>